Amazon.IonObjectMapper/IonSerializationFormat.cs

```diff
@@ -13,10 +13,24 @@
 
 namespace Amazon.IonObjectMapper
 {
+    /// <summary>
+    /// Format for serializing/deserializing Ion.
+    /// </summary>
     public enum IonSerializationFormat
     {
+        /// <summary>
+        /// Binary format for serializing/deserializing Ion.
+        /// </summary>
         BINARY,
+
+        /// <summary>
+        /// Text format for serializing/deserializing Ion.
+        /// </summary>
         TEXT,
+
+        /// <summary>
+        /// Pretty text format for serializing/deserializing Ion.
```

Comment:
Can we provide a code example to show the difference between TEXT and PRETTY_TEXT?

Reply:
Addressed.